src/cacheout/cache.py

```diff
@@ -462,10 +462,6 @@ def get_ttl(self, key: t.Hashable) -> t.Optional[T_TTL]:
                 return None
 
             ttl = expire_time - self.timer()
-            if ttl <= 0:
```

Comment:
I delete this case, because if the key has expired, `self._has(key)` will delete the key and return False. So `ttl <= 0` will never be True.

Reply:
Good point!